tools/schema_registry_tools/schemas_command_generator.py

```diff
@@ -43,10 +124,17 @@ def generate_options():
         parser = argparse.ArgumentParser(
             description='Redpanda Schema Registry _schemas Command Generator')
         parser.add_argument('path', type=str, help='Path to the file')
+        parser.add_argument('-t',
+                            '--test',
+                            action='store_true',
+                            help='Run tests')
         return parser
 
     parser = generate_options()
     options, _ = parser.parse_known_args()
+    if options.test:
+        run_tests()
+        exit(1)
```

Comment:
I think the exit status should be 0 for success?

Reply:
Right. Fixed in 2429790.